File: CodingAcademy_Programs/MongoDB/update.rb
require 'rubygems'
require 'mongo'

include Mongo

$client = Mongo::Client.new(['127.0.0.1:27017'], database: 'LXF')
Mongo::Logger.logger.level = ::Logger::ERROR
$collection = $client[:someData]

# Updating documents with the 'update_many()' function
result = $collection.find('n' =>
                               { '$gt' => 205_000 }).update_many(
                                 '$inc' => { code: 10 }
                               )
# The next command returns the number of documents,
# that were updated
puts result.n

File: CodingAcademy_Programs/MongoDB/update.py
#!/usr/bin/python3
from pymongo import MongoClient

client = MongoClient("127.0.0.1", 27017)
db = client.LXF

# Choose a collection
someData = db.someData
for document in someData.find():
    print(document)

for doc in someData.find({"code": {"$gt": 30}}).sort("n"):
    print(doc)

# Choosing a Document
print(someData.find_one({"n": 1}))

# Update it
someData.update({"n": 1}, {'$set': {'newField': 10}})

# Print again
print(someData.find_one({'newField': 10}))
